rrrspec-client/lib/rrrspec/client/slave_runner.rb

```diff
@@ -135,9 +140,15 @@ def example_failed(notification)
             end
           end
 
+          def dump_summary(notification)
+            @errors_outside_of_examples_count = notification.errors_outside_of_examples_count
```

Comment:
Isn't it necessary to be `+=`?

Reply:
To be more precise,

1. SlaveRunner takes one task (== spec file)
2. Call RSpecRunner#reset
    - Clear result of the previous task
3. Call RSpecRunner#setup
    - Load spec files
4. Call RSpecRunner#run
    - Run each example
5. Store result of the task

So `dump_summary` is called once for each spec file.